docs/sphinx/reference-frontend-api.rst

```diff
@@ -239,6 +239,14 @@ Structures/Enumerations
 
    Undo redo callback
 
+.. type:: video_t *(*multitrack_video_start_cb)(void *param)
```

Comment:
It probably makes sense to include the view name in both of these callbacks.

Reply:
changed in https://github.com/obsproject/obs-studio/pull/10635#commits-pushed-f65594f

Reply:
It looks like the code was changed, but not the documentation.

Reply:
@RytoEX I just pushed a fixup commit with documentation changes.